tests/test_core/test_core_destroy.py

```diff
@@ -10,49 +13,50 @@
 CORE_FUNCTION_STUB = "matcha_ml.core.core"
```

Comment:
Nit: the stub isn't being used anymore so far as I can see. We could use it, or remove it.